docs/privacyguides/faq.md

```diff
@@ -1,75 +1,59 @@
+- As a non-profit, we strictly cannot use donations to support political campaigns/candidates or attempt to influence legislation. Earnings also will not inure to the benefit of any private shareholder or individual.
 
-## How much of my donation goes towards individuals and projects supported by the Monero Fund?
+## Are there other ways to support Privacy Guides?
 
-- We set up campaigns to closely match the amount needed to pay these individuals and projects. There are some credit card processing fees and other related fees that we try to estimate. Typically, we add a buffer of 3% to the project campaign to account for these.
-- If there are any proceeds are retained by the MAGIC Monero Fund, and will be used on related projects.
-- If a campaign fails to raise the full amount of funds, then the committee will work with the intended recipient to revise the scope of the project, or the raised funds will be used on related projects.
-
-## Can an individual or project be funded by both the Monero Community Crowdfunding System (CCS) and the Monero Fund?
-
-- Yes. There is no expectation of exclusivity, but a proposer cannot raise funds elsewhere for the exact same work that a MAGIC Monero Fund proposal is intended to cover. This allows a proposer to maintain multiple, simultaneous funding options, so long as the scope of work included in the proposal is unique.
-
-## Do you distribute grants in Monero, Bitcoin or fiat?
-
-- We distribute grants in the currency of the researchers choosing. We prefer to pay in XMR but will gladly distribute funds in BTC, USD, or other crypto or fiat currencies that are highly liquid.
+- We alternatively accept donations via [GitHub Sponsors](https://github.com/sponsors/privacyguides) if you prefer that platform or would like to publicize your support there.
+- Another option to support us is by buying our merchandise from [HelloTux](https://hellotux.com/privacyguides). We get a small commission for each item sold, and you get a quality product to show for it.
+- You can also support Privacy Guides via non-financial means by getting involved with our [community](https://discuss.privacyguides.net/), [helping edit the site](https://github.com/privacyguides/privacyguides.org), or by [contributing translations](https://crowdin.com/project/privacyguides).
 
 ## How do you collect Monero donations?
 
 - We use BTCPay Server to receive all cryptocurrency donations. For fiat donations, we leverage Stripe Payments.
 
-
 ## Can you provide the view key to verify the Fund's incoming Monero donations?
 
 - Yes. You can create a view-only Monero wallet with this information:
 - Main address:
   ```
-  4458gM5iAPuJBeVf5u9DwS4Hj82qNPzAcgD7wF9VrRCRb4M635UtmZNPPDonqXkBsb2RmKjUSRLXtJNZCjZgnNFKKW3SxBz
+  TODO
   ```
 - Secret view key:
   ```
-  2595ab540844231f66c2266f12b174461f81c09b39010dc42697a6a58a7d680c
+  TODO
   ```
 - Restore height: 2724728
```

Comment:
@SamsungGalaxyPlayer do you mind adding this information please?

Reply:
This is the wallet for the entire website, not specific to this fund:

4BE3G7JCwcphuZYGw56AcpDEsDsZ7kCDjZwkxQSf3cSrJxbq4Mgqu1tNZeQny1QUZG9aeNWaHEiK82fvzsCcuySJSFccM5u

9a7202066d71e2b72d5fe61baf2f77a91258387b2abdf2516797f3ce30400802